### PR TITLE
Improved documentation, and code.

### PR DESCRIPTION
**Pull request by malishahi**
Made a few improvements in order to make it work.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ function Strategy(options, verify) {
   this.options = options;
   this._base = Object.getPrototypeOf(Strategy.prototype);
   this._base.constructor.call(this, this.options, verify);
-  this.name = 'Keycloak';
+  this.name = 'keycloak';
```
Comment (svarlamov) [inline index.js:23]: Is this potentially a breaking change -- or would this continue to work for existing implementations?

**Review by svarlamov — COMMENTED** [review]
Thanks for the contribution @malishahi -- really appreciated! Can you let me know about the capitalization change I commented on in the code? Thanks 😄

File: index.js (hunk 1 above)
Comment (malishahi) [reply index.js:23]: Sounds good. This will continue to work with existing implementations.

File: README.md [hunk 2]
```diff
@@ -62,15 +68,49 @@ passport.use(
...
-);
-router.get(
-  routes.AUTH_KEYCLOAK_CALLBACK,
-  passport.authenticate("keycloak", DEFAULT_PASSPORT_OPTIONS),
-  AuthController.keyCloakSuccess
+app.get('/api/auth/keycloak/',
+    passport.authenticate("keycloak", (err, user) => {
+        if (err || !user) {
+            console.log(err, user);
+            res.json({ err, user });
```
Comment (Irrelon) [inline README.md:75]: res is not defined here!

**Pull request closed without merging**